src/wp-includes/html-api/class-wp-html-processor-state.php

```diff
@@ -215,6 +227,31 @@ class WP_HTML_Processor_State {
 	 */
 	const INSERTION_MODE_IN_TEMPLATE = 'insertion-mode-in-template';
 
+	/**
+	 * No-quirks mode document compatability mode.
+	 *
+	 * > In no-quirks mode, the behavior is (hopefully) the desired behavior
+	 * > described by the modern HTML and CSS specifications.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @var string
+	 */
+	const NO_QUIRKS_MODE = 'no-quirks-mode';
+
+	/**
+	 * Quirks mode document compatability mode.
+	 *
+	 * > In quirks mode, layout emulates behavior in Navigator 4 and Internet
+	 * > Explorer 5. This is essential in order to support websites that were
+	 * > built before the widespread adoption of web standards.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @var string
+	 */
+	const QUIRKS_MODE = 'quirks-mode';
```

Comment:
So far I haven't seen any implications for limited-quirks mode. Most language reads like "if document is in quirks mode".

I'll double-check, but I think limited quirks comes specifically in the fragment case.

Reply:
after another scan through the spec, it does seem that limited-quirks mode behaves just like no-quirks mode, and the only impact this has is on case-sensitivity in matching IDs and class names.

I don't see a reason to differentiate them, especially since browsers don't. the name `CSS1Compat` makes more sense to me now too, since it's really about how CSS selectors behave